website/docs/docs/collaborate/documentation.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Documentation"
+title: "About documentation"
```

Comment:
It seems like most of the section titles are being shortened and/or streamlined but this one is adding content.  Just curious why we're using "About" here when the language is being removed from other areas

Reply:
@matthewshaver : it just kinda naturally evolved from working on the IA project. it's not all over the place but is there a few times: 

![Screen Shot 2022-10-04 at 12 58 54 PM](https://user-images.githubusercontent.com/107218380/193914283-b41fbb74-d32d-46c5-a303-a00a057917bc.png)

i don't have strong feels to keep "About". personally, it read it bit better for me. but am open to suggs! i'm pro consistency.